frontend/src/views/camp/Collaborators.vue

```diff
@@ -56,7 +56,6 @@ Displays collaborators of a single camp.
                 <e-text-field
                   v-model="inviteEmail"
                   :name="$tc('views.camp.collaborators.email')"
-                  vee-rules="email"
```

Comment:
Why was this removed?

Reply:
oops, shouldn't. Test code to trigger backend errors. Shall I put it back or will you?

Reply:
i will put it back, no problem.